packages/react/src/page-alert/PageAlertTitle.tsx

```diff
@@ -4,21 +4,24 @@ import { Text } from '../text';
 
 export type PageAlertTitleProps = PropsWithChildren<{
 	as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+	/** @deprecated use `hasCloseButton` instead */
 	hasDismissButton?: boolean;
+	hasCloseButton?: boolean;
 }>;
 
 export const PageAlertTitle = ({
 	as = 'h3',
 	children,
 	hasDismissButton,
+	hasCloseButton,
```

Comment:
Should `console.warn` this is deprecated and also add this as a new change in the changeset

Reply:
warning if hasCloseButton is used now